app/views/katello/api/v2/content_facet/base.json.rabl

```diff
@@ -0,0 +1,11 @@
+
+attributes :content_view_id, :content_view_name
+attributes :lifecycle_environment_id, :lifecycle_environment_name
```

Comment:
These almost seem better as a node, however, if these are the param names you have to pass in when doing something like an update, I can see leaving them split.